src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -20,6 +18,7 @@ public class CodeGenIL:IDisposable
         private ILGenerator ilGen;
         internal string className;
         internal string methodName;
+        private bool isInternal; 
```

Comment:
maybe deserves a comment?

Reply:
I think `builtin` would be a better name as it is used currently in the VM.

Reply:
Both are used in the current VM but I think Internal is the most common. For example, the method for checking for these are `CoreUtils.IsInternalMethod()`. But I am OK changing to BuiltIn.